src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -327,6 +335,8 @@ void ContextAttribute::present(const std::string& indent, int ix)
     compoundValueP->show(indent + "  ");
   }
 
+  PRINTF("%s  PA:       %s\n", indent.c_str(), providingApplication.c_str());
+
```

Comment:
I should have a macro somewhere ...
Doing this:

X(x) (x == true)? "true" : "false"